x-pack/plugins/maps/public/reducers/non_serializable_instances.d.ts

```diff
@@ -7,21 +7,49 @@
 import { Adapters } from 'src/plugins/inspector/public';
 import { AnyAction } from 'redux';
 
+export interface ResultMeta {
+  featuresCount?: number;
```

Comment:
Would this just default to `0` vs. not be included? i.e.- can we just make it `featuresCount: number;`?

Reply:
The event handler is called for vector and non-vector layer loading (like TileLayer loading). I think it would be misleading to return featuresCount for non-vector layers.